jujugui/static/gui/src/app/components/generic-input/generic-input.js

```diff
@@ -51,11 +54,15 @@ YUI.add('generic-input', function() {
     /**
       Validate the field value.
 
+      @param {Object} evt The trigger event.
       @method validate
     */
-    validate: function() {
+    validate: function(evt) {
       const validate = this.props.validate;
       if (!validate) {
+        if (this._onKeyUp) {
```

Comment:
What's this._onKeyUp?
Should this whole block be replaced by the call to this. _keyUpHandler?